### ISSUE TITLE
Various things in the new 'quickstart.sh' script

### ISSUE THREAD

**Issue by peterkaminski** [GovReady/govready-q#705]
Some observations after using and working on the `quickstart.sh` script.

### Spurious-ish user from `manage.py first_run`

`quickstart.sh` runs `manage.py first_run`, which asks you to create an initial superuser.  If you generate test data, that superuser isn't in the test organization.  That initial user is sort of a waste of time to create.

I wonder if switching the order in which quickstart runs `manage.py first_run` and `manage.py populate` might suppress the initial superuser creation.

### Exiting quickstart sort of doesn't

Pressing "n" at the "run the webserver" question sort of hangs instead of exiting cleanly.

(I think something similar happens with "y", but it happens the web server is running and you've killed it, so it doesn't matter as much.)

### "Cannot answer question of type..."

There are lots of "Cannot answer question of type..." warnings when generating test data.  Not sure if this is a problem or not.

### Running quickstart again

* Another organization is added.  I think that's good, but I'm not sure.  (It could detect the previous test, and ask if it should make another org or not.)
* Not a big deal, but it's cool how it suppresses some steps, but `fetch-vendor-resources.sh` always runs.  We could hack `fetch-vendor-resources.sh` to detect existing downloads and possibly skip downloading again.

**Comment by ethankaminski** [comment]
I've opened PR #708 for the first two points here - good feedback, and easy to action.

The "Cannot answer question of type..." is a little more involved, so I'll handle that in a separate pull request. (possibly two, since that's something that also needs to land in the 0.9.0 code)

With `fetch-vendor-resources.sh`, that suggestion does sounds safely doable; but when I tried it, I observed that the script was aggressively removing old data when run. That seems like a useful measure for having repeatable builds, and given that `pip3 install` is also fairly slow, I myself don't see the need to build a local cache for the vendor files. If that particular script is a significant bottleneck, though, I do know how it could be done, while retaining the current reset-to-clean-slate approach.
i.e., I'm going to skip that for now.

Second organization on rerun: since this is skippable by answering "no", that seems like it's okay.
